contracts/scripts/native_solc_compile_all_ccip

```diff
@@ -38,6 +39,13 @@ compileContract () {
       --bin-runtime --hashes --metadata --metadata-literal --combined-json abi,hashes,metadata,srcmap,srcmap-runtime \
       --evm-version paris \
       "$ROOT"/contracts/src/v0.8/"$1"
+  
+  zksolc --overwrite -O3 --metadata-hash none \
```

Comment:
is the overwrite same as the "mode"?
& any other option to provide "fallback_to_optimizing_for_size"?

Reply:
```
--fallback-Oz                 Try to recompile with -Oz if the bytecode is too large
```
i think unless you dont specify this flag, it should be false.
`--overwrite` is for overwriting a file if the wrapper already exists